internal/lsp/completions/providers/policy.go

```diff
@@ -70,20 +68,15 @@ func (p *Policy) Run(
 	inputContext["path_separator"] = string(os.PathSeparator)
 
 	workspacePath := uri.ToPath(opts.ClientIdentifier, opts.RootURI)
-	inputDotJSONPath, inputDotJSONReader := rio.FindInput(
+
+	inputDotJSONPath, inputDotJSONContent := rio.FindInput(
```

Comment:
Not sure if it's worth saving some future confusion with the names here.
```suggestion
	inputDocumentFilePath, inputDocumentFileContent := rio.FindInput(
```